content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -275,7 +275,7 @@ that it applies to all Ingress, such as the load balancing algorithm, backend
 weight scheme, and others. More advanced load balancing concepts
 (e.g. persistent sessions, dynamic weights) are not yet exposed through the
 Ingress. You can still get these features through the
-[service loadbalancer](https://github.com/kubernetes/ingress-nginx/blob/master/docs/ingress-controller-catalog.md).
+[service loadbalancer](https://github.com/kubernetes/ingress-nginx).
```

Comment:
@aledbf As author of https://github.com/kubernetes/ingress-nginx/pull/2778, could you review this one-line change, please? Is this a correct fix?

Thanks!

Reply:
@php-coder yes, this is correct